pkgs/tools/misc/ttygif/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "ttygif";
+  version = "1.4.0";
+
+  src = fetchurl {
```

Comment:
The fetchFromGitHub helper is usually prefered. It's especially helpful if we want to select commits off master.
```
  src = fetchFromGitHub {
    owner = "icholy";
    repo = pname;
    rev = version;
    sha256 = "0apf7j6291rgv4s7c58cjq284ldc3scsws0003wdqayyr9fxr8vc";
  };
```